### PR TITLE
converted all single line comments to multiple line comments

### PR DESCRIPTION
**Pull request by Augustinetharakan12**

### CODE REVIEW

File: django/template/response.py [hunk 1]
```diff
@@ -12,34 +12,39 @@ class SimpleTemplateResponse(HttpResponse):
...
     def __init__(self, template, context=None, content_type=None, status=None,
                  charset=None, using=None):
-        # It would seem obvious to call these next two members 'template' and
-        # 'context', but those names are reserved as part of the test Client
-        # API. To avoid the name collision, we use different names.
+        """
+        It would seem obvious to call these next two members 'template' and
+        'context', but those names are reserved as part of the test Client
+        API. To avoid the name collision, we use different names.
+        """
```
Comment (carltongibson) [inline django/template/response.py]: This converts the inline comment to the docstring for the method. That's not correct. Please revert.

File: django/template/response.py [hunk 2]
```diff
@@ -12,34 +12,39 @@ class SimpleTemplateResponse(HttpResponse):
...
 
         self._post_render_callbacks = []
-
-        # _request stores the current request object in subclasses that know
-        # about requests, like TemplateResponse. It's defined in the base class
-        # to minimize code duplication.
-        # It's called self._request because self.request gets overwritten by
-        # django.test.client.Client. Unlike template_name and context_data,
-        # _request should not be considered part of the public API.
+        """
```
Comment (carltongibson) [inline django/template/response.py]: This is meant to be an inline comment, not a multi-line string. Please revert.

File: django/template/response.py [hunk 3]
```diff
@@ -12,34 +12,39 @@ class SimpleTemplateResponse(HttpResponse):
...
+        It's called self._request because self.request gets overwritten by
+        django.test.client.Client. Unlike template_name and context_data,
+        _request should not be considered part of the public API.
+        """
         self._request = None
-
-        # content argument doesn't make sense here because it will be replaced
-        # with rendered template so we always pass empty string in order to
-        # prevent errors and provide shorter signature.
+        """
```
Comment (carltongibson) [inline django/template/response.py]: As above.

File: django/template/response.py [hunk 4]
```diff
@@ -12,34 +12,39 @@ class SimpleTemplateResponse(HttpResponse):
...
+        prevent errors and provide shorter signature.
+        """
         super().__init__('', content_type, status, charset=charset)
-
-        # _is_rendered tracks whether the template and context has been baked
-        # into a final response.
-        # Super __init__ doesn't know any better than to set self.content to
-        # the empty string we just gave it, which wrongly sets _is_rendered
-        # True, so we initialize it to False after the call to super __init__.
+        """
```
Comment (carltongibson) [inline django/template/response.py]: As above

File: django/template/response.py [hunk 5]
```diff
@@ -58,7 +63,9 @@ def __getstate__(self):
         return obj_dict
 
     def resolve_template(self, template):
-        """Accept a template object, path-to-template, or list of paths."""
+        """
```
Comment (carltongibson) [inline django/template/response.py]: This was fine as a single line no?

File: django/template/response.py [hunk 6]
```diff
@@ -84,8 +92,8 @@ def rendered_content(self):
         return content
 
     def add_post_render_callback(self, callback):
-        """Add a new post-rendering callback.
-
+        """
+        Add a new post-rendering callback.
```
Comment (carltongibson) [inline django/template/response.py]: Needs a blank line below to maintain the spacing.

File: django/template/response.py [hunk 7]
```diff
@@ -95,10 +103,9 @@ def add_post_render_callback(self, callback):
             self._post_render_callbacks.append(callback)
 
     def render(self):
-        """Render (thereby finalizing) the content of the response.
-
+        """
+        Render (thereby finalizing) the content of the response.
```
Comment (carltongibson) [inline django/template/response.py]: Needs a blank line below to maintain the spacing.

**Comment by timgraham** [comment]
I agree with Carlton that changes aren't really needed here.

**Pull request closed without merging**